app/initializers/customizations.js

```diff
@@ -0,0 +1,9 @@
+import Checkbox from '@ember/component/checkbox'
+
+export default {
+  initialize: function() {
+    Checkbox.reopen({
+      attributeBindings: ['data-test-category']
+    })
+  }
+}
```

Comment:
Why are you doing this?

Reply:
To define custom attributes in the input helper, you need to define them. It would be for the acceptance test.